ui/apps/platform/src/Containers/Violations/ViolationsTablePage.tsx

```diff
@@ -46,6 +63,7 @@ function ViolationsTablePage(): ReactElement {
     const { page, perPage, setPage, setPerPage } = useURLPagination(50);
 
     // Handle changes in the currently displayed violations.
+    const [isLoadingAlerts, setIsLoadingAlerts] = useState(false);
```

Comment:
Since this Violations table isn't using TBodyUnified yet, I still want to have some indication that things are changing when we switch tabs. I decided to show a spinner that goes away once the data is updated. This works for the use case. The goal is to limit the scope increase from improvements in the search filters. Using TBodyUnified will involve figuring out how to deal with polling updates (constant spinners in the table). We will need to figure out how to deal with that in TBodyUnified before we decided to use it here.

Reply:
This makes sense short term for sure. Regarding the TBodyUnified, my gut reaction is that we shouldn't deal with the polling state directly in that component. I think we need to bring the topic up with UX as there are some rough edges to the current implementation. 

e.g. If you are anywhere other than the first page of results, any new updates will shift the order of what is displayed on screen so the data might be changing constantly. Actually this happens on the first page as well, but it is more of an issue once the user wants to browser through the pages. Otherwise it is a relatively bad experience to "browse forward" since the data is changing out from underneath you and it isn't clear when you miss viewing certain rows.